### ISSUE TITLE
[BUG] "Select image number from results for copying" doesn't work neither in txt2img nor img2img

### ISSUE THREAD

**Issue by Alphyn-gunner** [hlky/stable-diffusion#66]
![image](https://user-images.githubusercontent.com/34932866/187075869-b56f8e4e-17ca-4b3d-90c4-118d1122abff.png)
The text field that was used to select the image to copy to img2img is not editable anymore and it's impossible to select an image from the batch.

**Comment by Alphyn-gunner** [comment]
Ok, after some testing: 

In txt2img mode:

- Of the 3 buttons at the top, only the first one (Copy to clipboard) works. It copies the image currently selected by clicking on it.
- The second and the third button (Push to img2img, Upscale w/ ESRGAN) do nothing.
- The text field is unresponsive
- The bottom "Push to img2img" works and sends the image currently selected by clicking on it.

In img2img:

- There's only one button, it always sends the first image, the text field is not editable.

**Comment by hlky** [comment]
Is this issue from the stable version or the development version? 

If you have followed the instructions of where to put the issue thank you but could you still please state that you are using the stable version so I know you have followed the instruction.

If this is from the dev version I will move it to the dev repo

**Comment by butch2k** [comment]
Problem is with the stable version, i have the same one as the OP. I did not check the dev branch.

**Comment by Alphyn-gunner** [comment]
Yes, it's the stable version, that's  why I created the issue on this repo.

**Comment by hlky** [comment]
Ok, so have you considered trying the dev version? I've just accepted some changes from a couple of hours ago to be synced to stable anyway, I think those changes fixed it

**Comment by butch2k** [comment]
ok i'll have a look

**Comment by Alphyn-gunner** [comment]
I've just tested the dev version. Same exact behavior as I described in the second post.

**Comment by butch2k** [comment]
![image](https://user-images.githubusercontent.com/961507/187077934-09513d48-bc16-4475-8587-92d9c7a60267.png)
The field is still disabled, number does not change when selecting an image nor does it permit any input, the button "Push to img2img" send the correct manualy selected image to img2img screen though.

**Comment by butch2k** [comment]
Copy to clipboard work, might worth a message somewhere as a feedback to the copy operation.
The "push to img2img" next to it and the "Upscale w/ESRGAN" still don't do anything.
"Upscale w/ESRGAN" css is missing the  border-radius: 0px !important; btw.
